src/main/java/de/tum/cit/aet/core/domain/Document.java

```diff
@@ -0,0 +1,32 @@
+package de.tum.cit.aet.core.domain;
+
+import de.tum.cit.aet.application.domain.CustomField;
+import de.tum.cit.aet.core.domain.constants.DocumentType;
+import de.tum.cit.aet.usermanagement.domain.Applicant;
+import jakarta.persistence.*;
+import java.util.UUID;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "documents")
+public class Document {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID documentId;
+
+    @ManyToOne
+    @JoinColumn(name = "applicant")
+    private Applicant applicant;
+
+    private DocumentType type;
+
+    private String documentPath;
+
+    @ManyToOne
+    @JoinColumn(name = "custom_field")
```

Comment:
(nit pick) I suggest calling it custom_field_id to stay consistent with the rest of the naming schemes of the other join columns
```suggestion
    @JoinColumn(name = "custom_field_id")
```